contrib/src/templates/context.rs

```diff
@@ -45,7 +48,8 @@ impl Context {
             }
         }
 
-        Engines::init(&templates).map(|engines| {
+        Engines::init(&templates).map(|mut engines| {
+            customize_callback(&mut engines);
```

Comment:
We should call the user's function after initialization. That way, we don't have to pass it in to `initialize` at all, and `Context` can stay ignorant that it even exists.